### DISCUSSION TITLE
How to start

### DISCUSSION THREAD

**Discussion by alwaysmavs** [oomol-lab/support#1] [General]
If you are using it for the first time and don’t know how to get started, please ask here！

**Comment by 836145715** [comment]
想问一下，这个软件是一个人开发的吗，太厉害了，我脑海里一直有这种软件的构思，但是技术达不到做不出来，第一次看到有这样的实现，真的是非常惊喜，这个对编程开发者非常的友好

**Reply by alwaysmavs** [reply]
我们是一个创业公司，主要都是研发。所以想做个自己喜欢用的产品。

**Reply by Xuanfq** [reply]
请问公司有多少人在focus这个项目的开发？

**Reply by BruceLeeX** [reply]
我怎么一直是“启动oomol虚拟机”？一直这个样子！！

**Reply by winomdlb** [reply]
一样，请问如何解决

**Comment by 836145715** [comment]
是否有交流群可以加入呢

**Reply by alwaysmavs** [reply]
![oomol](https://github.com/user-attachments/assets/21341c68-d9c3-471b-a3c7-d361bd1c8f05)
可以加我们企微群

**Comment by Ainesder** [comment]
OOMOOL studio访问文件的时候没有显示本地磁盘,有没有人知道该怎么办？
